### ISSUE TITLE
Spelare - transfer - aktuellt bud

### ISSUE THREAD

**Issue by VilijamRigo** [kvarak/htstatus#27]
Skulle man kunna stoppa in en kolumn i slutet för att se om aktuellt bud på en spelare man säljer?

**Comment by kvarak** [comment]
Jo, men det är en helt annan data som ska hämtas då. Vi kan lägga till detta när vi håller på med transfern sen.

**Comment by kvarak** [comment]
When https://framagit.org/Pierre86/pychpp/-/issues/14 is fixed, I can do this.

**Comment by kvarak** [comment]
The fix exists in pychpp 0.2.11. This is no longer blocked.
